pkg/kubectl/cmd/rollout/rollout.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	rollout_long    = `rollout manages a deployment using subcommands like "kubectl rollout undo deployment/abc"`
+	rollout_long    = `kubectl rollout manages a deployment using subcommands like "kubectl rollout undo deployment/abc"`
```

Comment:
"kubectl rollout" isn't necessary, you can just start with "Manages a deployment ..."? It looks like that's what we are doing with other commands.